netcode.c

```diff
@@ -4163,7 +4207,12 @@ void netcode_server_connect_client( struct netcode_server_t * server,
     char address_string[NETCODE_MAX_ADDRESS_STRING_LENGTH];
 
     /* TODO: try to find a way to deliver match itd. */
-    if ( !skillz_add_client_to_match( server, 111, client_id ) )
+    int testId = 0;
+    if( server->num_connected_clients >= 3 )
+        testId = 222;
+    else
+        testId = 111;
```

Comment:
what are these testIds for?

Reply:
This is because there is no delivery system for actual match_ids.  These allow for up to 2 matches to be made.